main-app/app/scripts/providers/audio.js

```diff
@@ -0,0 +1,28 @@
+(function () {
+    'use strict';
+
+    angular.module('Tombola.NoughtsAndCrosses.Providers')
+        .service('Audio', function () {
+            var audio = function(){
```

Comment:
Ctor functions need are PascalCased - surprised your linter didn't pick this up.
The problem with "Audio" is it'll conflict with the instantiation of the native Audo objects below - Stack Overflow?